com.unity.render-pipelines.core/Editor/Lighting/ProbeVolume/ProbeVolumeUI.Skin.cs

```diff
@@ -12,6 +12,7 @@ internal static class Styles
 
             internal static readonly GUIContent s_Size = new GUIContent("Size", "Modify the size of this Probe Volume. This is independent of the Transform's Scale.");
             internal static readonly GUIContent s_DebugColorLabel = new GUIContent("Debug Color", "This color is used to visualize per-pixel probe volume weights in the render pipeline debugger.");
+            internal static readonly GUIContent s_MinMaxSubdivSlider = new GUIContent("Subdivision Controller", "TODO");
```

Comment:
Can we give a tooltip or are we waiting for artists feedback?

Reply:
I'll add one, but it'll probably change since artists didn't validated this approach